docs/fsharp-cheatsheet.md

```diff
@@ -67,88 +68,88 @@ Other common examples are `F` or `f` for 32-bit floating-point numbers, `M` or `
+Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to create a symbol with spaces in it to improve readability, especially for unit testing:
 
-	let ``square, negate, then print`` x = 
-		x |> square |> negate |> print
-
-This operator is essential in assisting the F# type checker by providing type information before use:
+    let ``square, negate, then print`` x = 
+        x |> square |> negate |> print
+	
+This operator [what operator?] is essential in assisting the F# type checker by providing type information before use:
 
     let sumOfLengths (xs : string []) = 
-		xs 
-		|> Array.map (fun s -> s.Length)
-		|> Array.sum
+        xs 
+        |> Array.map (fun s -> s.Length)
+        |> Array.sum
 
-Composition operator `>>` is used to compose functions:
+Composition operator `>>` is used to compose functions in a way that's more more concise than shown above. Also, appending a backtick to a symbol is a conventional way to mark the new symbol as a variant of a previously defined symbol. It's like appending `1` but cooler:
 
-	let squareNegateThenPrint' = 
-		square >> negate >> print
+    let squareNegateThenPrint' = 
+        square >> negate >> print
   
 ### Recursive functions
 The `rec` keyword is used together with the `let` keyword to define a recursive function:
 
-	let rec fact x =
-	    if x < 1 then 1
-	    else x * fact (x - 1)
+    let rec fact x =
+        if x < 1 then 1
+        else x * fact (x - 1)
 
-*Mutually recursive* functions (those functions which call each other) are indicated by `and` keyword:
+*Mutually recursive* functions (functions that call each other) are indicated by `and` keyword:
 
-	let rec even x =
-	   if x = 0 then true 
-	   else odd (x - 1)
+    let rec even x =
+        if x = 0 then true 
+        else odd (x - 1)
 
-	and odd x =
-	   if x = 0 then false
-	   else even (x - 1)
+     and odd x =
+        if x = 0 then false
+        else even (x - 1)
 
 <a name="PatternMatching"></a>Pattern Matching
 ----------------
 Pattern matching is often facilitated through `match` keyword.
 
-	let rec fib n =
-	    match n with
-	    | 0 -> 0
-	    | 1 -> 1
-	    | _ -> fib (n - 1) + fib (n - 2)
+    let rec fib n =
+        match n with
+        | 0 -> 0
+        | 1 -> 1
+        | _ -> fib (n - 1) + fib (n - 2)
 
-In order to match sophisticated inputs, one can use `when` to create filters or guards on patterns:
+More sophisticated matching can be done with `when`, to create a filter or guard on a pattern:
 
-	let sign x = 
-		match x with
-	    | 0 -> 0
-	    | x when x < 0 -> -1
-	    | x -> 1
+    let sign x = 
+        match x with
+        | 0 -> 0
+        | x when x < 0 -> -1
+        | x -> 1
 
-Pattern matching can be done directly on arguments:
+If matching is the first thing a function will do, use the `function` keyword to start the match:
```

Comment:
I don't think this change adds clarity, as "matching is the first thing a function will do" in all examples here, not just the example with `function`.

Reply:
Maybe I'‘m missing something.  It seems like an alternate way of doing the same thing. Which is preferred?

Reply:
Its just an alternate syntax, and some people frown on it as it doesnt' save any chars (just one line) and can arguably be harder to scan.
As noted below, convey that its identical by just doing `sign` as a `function`